### ISSUE TITLE
Dynamically adding new devices to a Bridge failing

### ISSUE THREAD

**Issue by JimBuzbee** [project-chip/matter.js#197]
A dynamically added device to an already running Bridge is not recognized by either Apple or Google. Log of action follows:

2023-07-09 07:59:21.286 DEBUG MessageExchange New exchange protocol: 1 id: 39191 active retransmit ms: 300 idle retransmit ms: 5000 retries: 5
2023-07-09 07:59:21.287 DEBUG MessageExchange Message « id: 39621/39191/218056380 type: 1/8 reqAck: true payload: 1528002801360215370024000224010624020118350118181824ff0118
2023-07-09 07:59:21.287 DEBUG InteractionProtocol Received invoke request from udp://fe80::e6f0:42ff:fe3d:67b2%en0:5540 on session secure/39621: MA-onofflight(0x2)/On/Off(0x6)/on(0x1)
2023-07-09 07:59:21.288 DEBUG CommandServer Invoke on with data undefined
2023-07-09 07:59:21.288 DEBUG InteractionProtocol Storing attribute onOff in cluster On/Off (6)
2023-07-09 07:59:21.295 DEBUG CommissioningServer Endpoint structure got updated ...
2023-07-09 07:59:21.299 DEBUG CommissioningServer Assigned endpoint id 6 for endpoint with serial_node-matter-1688867942490-index_0-serial_node-matter4 / device MA-onofflight and stored it
2023-07-09 07:59:21.303 DEBUG InteractionProtocol Restoring attribute breadcrumb (0) in cluster breadcrumb (48)
2023-07-09 07:59:21.304 DEBUG InteractionProtocol Restoring attribute onOff (0) in cluster onOff (6)
2023-07-09 07:59:21.304 DEBUG CommandServer Invoke on response : undefined
2023-07-09 07:59:21.304 DEBUG ExchangeManager Message » id: 39621/39191/3083325674 type: 1/9 acked: 218056380 reqAck: true payload: 152800360115350137002400022401062402011835012400001818181824ff0118
2023-07-09 07:59:21.320 DEBUG MessageExchange Message « id: 39621/39191/218056381 type: 0/16 acked: 3083325674 reqAck: false payload:
2023-07-09 07:59:21.344 DEBUG SubscriptionHandler Sending subscription update message for ID 1859306161 with 1 values
2023-07-09 07:59:21.344 DEBUG MessageExchange New exchange protocol: 1 id: 12658 active retransmit ms: 300 idle retransmit ms: 5000 retries: 5
2023-07-09 07:59:21.344 DEBUG SubscriptionHandler Sending subscription changes for ID 1859306161: [{"path":{"endpointId":2,"clusterId":6,"attributeId":0},"schema":{},"version":3,"value":true}]
2023-07-09 07:59:21.344 DEBUG ExchangeManager Message » id: 11491/12658/3083325675 type: 1/5 reqAck: true payload: 152600b1c2d26e36011535012400033701240202240306240400182902181818280424ff0118
2023-07-09 07:59:21.344 DEBUG SubscriptionHandler Sending subscription update message for ID 1859306162 with 1 values
2023-07-09 07:59:21.344 DEBUG MessageExchange New exchange protocol: 1 id: 12659 active retransmit ms: 300 idle retransmit ms: 5000 retries: 5
2023-07-09 07:59:21.344 DEBUG SubscriptionHandler Sending subscription changes for ID 1859306162: [{"path":{"endpointId":2,"clusterId":6,"attributeId":0},"schema":{},"version":3,"value":true}]
2023-07-09 07:59:21.344 DEBUG ExchangeManager Message » id: 16097/12659/3083325676 type: 1/5 reqAck: true payload: 152600b2c2d26e36011535012400033701240202240306240400182902181818280424ff0118
2023-07-09 07:59:21.345 DEBUG SubscriptionHandler Sending subscription update message for ID 1859306163 with 1 values
2023-07-09 07:59:21.345 DEBUG MessageExchange New exchange protocol: 1 id: 12660 active retransmit ms: 300 idle retransmit ms: 5000 retries: 5
2023-07-09 07:59:21.345 DEBUG SubscriptionHandler Sending subscription changes for ID 1859306163: [{"path":{"endpointId":2,"clusterId":6,"attributeId":0},"schema":{},"version":3,"value":true}]
2023-07-09 07:59:21.345 DEBUG ExchangeManager Message » id: 54443/12660/3083325677 type: 1/5 reqAck: true payload: 152600b3c2d26e36011535012400033701240202240306240400182902181818280424ff0118
2023-07-09 07:59:21.360 DEBUG MessageExchange Message « id: 39620/12660/210563175 type: 1/1 acked: 3083325677 reqAck: true payload: 1524000024ff0118
2023-07-09 07:59:21.360 DEBUG ExchangeManager Message » id: 54443/12660/3083325678 type: 0/16 acked: 210563175 reqAck: false payload:
2023-07-09 07:59:21.369 DEBUG MessageExchange Message « id: 39621/12658/218056382 type: 1/1 acked: 3083325675 reqAck: true payload: 1524000024ff0118
2023-07-09 07:59:21.369 DEBUG ExchangeManager Message » id: 11491/12658/3083325679 type: 0/16 acked: 218056382 reqAck: false payload:
2023-07-09 07:59:21.369 DEBUG MessageExchange Message « id: 39619/12659/222356079 type: 1/1 acked: 3083325676 reqAck: true payload: 1524000024ff0118
2023-07-09 07:59:21.369 DEBUG ExchangeManager Message » id: 16097/12659/3083325680 type: 0/16 acked: 222356079 reqAck: false payload:

**Comment by adminy** [comment]
is this still the case?

**Comment by JimBuzbee** [comment]
> is this still the case?

Seems to be working well with Apple, but Google has issues recognizing the new device, taking a long time. May be Google issue

**Comment by Apollon77** [comment]
This was party fixed by #199, but to fully work (and maybe also work correct for Google I will next adjust subscription handling

**Comment by adminy** [comment]
I've the latest from master, running on apple, 

bridged works on initial setup. But afterwards its disconnected even though its running. Logs just say:

```
Device is already commissioned. Waiting for controllers to connect ...
2023-07-22 19:06:15.832 INFO CaseServer Received pairing request from udp://fe80::2:dcad:bc0:21b%wlan0:60940
2023-07-22 19:06:15.889 INFO CaseServer session 31571 resumed with udp://fe80::2:dcad:bc0:21b%wlan0:60940 for Fabric 798d01e4(index 2) and PeerNode 716be233079e39e


2023-07-22 13:36:52.606 ERROR InteractionProtocol Subscription subscription 2661728770 for Session 57035: Error while sending initial data reports: (128) Received error status: 128
2023-07-22 13:42:31.103 ERROR InteractionProtocol Subscription subscription 2661728771 for Session 57035: Error while sending initial data reports: (128) Received error status: 128
2023-07-22 14:13:29.580 ERROR InteractionProtocol Subscription subscription 3359506321 for Session 52546: Error while sending initial data reports: (128) Received error status: 128
2023-07-22 19:06:18.111 ERROR InteractionProtocol Subscription subscription 931752853 for Session 31571: Error while sending initial data reports: (128) Received error status: 128
```

![IMG_4523](https://github.com/project-chip/matter.js/assets/22717869/08eaaabb-32b2-4b3c-835e-3bd990de704d)

**Comment by adminy** [comment]
[Here are the debug logs](https://logpaste.com/5SNAgWvD)

**Comment by Apollon77** [comment]
fixed on main, soon 0.5.0
